nixos/tests/scrutiny.nix

```diff
@@ -0,0 +1,66 @@
+import ./make-test-python.nix ({ lib, ... }:
+
+{
+  name = "scrutiny";
+  meta.maintainers = with lib.maintainers; [ jnsgruk ];
+
+  nodes = {
+    machine = { self, pkgs, lib, ... }: {
+      services = {
+        scrutiny.enable = true;
+        scrutiny.collector.enable = true;
+      };
+
+      environment.systemPackages =
+        let
+          seleniumScript = pkgs.writers.writePython3Bin "selenium-script"
+            {
+              libraries = with pkgs.python3Packages; [ selenium ];
+            } ''
+            from selenium import webdriver
+            from selenium.webdriver.common.by import By
+            from selenium.webdriver.firefox.options import Options
+            from selenium.webdriver.support.ui import WebDriverWait
+
+            options = Options()
+            options.add_argument("--headless")
+            service = webdriver.FirefoxService(executable_path="${lib.getExe pkgs.geckodriver}")  # noqa: E501
+
+            driver = webdriver.Firefox(options=options, service=service)
+            driver.implicitly_wait(10)
+            driver.get("http://localhost:8080/web/dashboard")
+
+            wait = WebDriverWait(driver, 60)
+
+            assert len(driver.find_elements(By.CLASS_NAME, "mat-button-wrapper")) > 0
+            assert len(driver.find_elements(By.CLASS_NAME, "top-bar")) > 0
```

Comment:
Why are we looking for the class names here? Couldn't we match a string like "Drive health at a glance" or so?

Reply:
The problem is finding something meaningful to latch onto in the UI. Happy to change this to match that string though.

Reply:
Adjusted in latest commit to find text from two different page elements. Thanks for the suggestion!